From f6fa886ac8ca4b9581ec194383ee5db04bfa5e9e Mon Sep 17 00:00:00 2001
From: Jason Bedard <jason@aspect.dev>
Date: Tue, 22 Mar 2022 18:27:17 -0700
Subject: [PATCH] fix(tests): fix gazelle_generation_test usage outside gazelle
 repo

---
 internal/generationtest/generationtest.bzl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/internal/generationtest/generationtest.bzl internal/generationtest/generationtest.bzl
index 273df3f3..a5c3dee4 100644
--- a/internal/generationtest/generationtest.bzl
+++ internal/generationtest/generationtest.bzl
@@ -38,9 +38,9 @@ def gazelle_generation_test(name, gazelle_binary, test_data, build_in_suffix = "
     """
     go_test(
         name = name,
-        srcs = ["//internal/generationtest:generation_test.go"],
+        srcs = [Label("//internal/generationtest:generation_test.go")],
         deps = [
-            "//testtools",
+            Label("//testtools"),
             "@io_bazel_rules_go//go/tools/bazel:go_default_library",
             "@in_gopkg_yaml_v2//:yaml_v2",
         ],
